src/Code.gs

```diff
@@ -307,6 +311,34 @@ const ChatGPTApp = (function () {
         return this;
       };
 
+      /**
+       * OPTIONAL
+       * 
+       * Enable a thread run with an OpenAI assistant.
+       * @param {string} assistantId - your assistant id
+       * @param {string} vectorStoreDescription - a small description of the available knowledge from this assistant
+       * @returns {Chat} - The current Chat instance.
+       */
+      this.retrieveKnowledgeFromAssistant = function (assistantId, vectorStoreDescription) {
+        assistantIdentificator = assistantId;
+        vectorStore = vectorStoreDescription;
+        return this;
+      }
+
+      /**
+       * OPTIONAL
+       * 
+       * Enable a thread run with an OpenAI assistant.
+       * @param {string} assistantId - your assistant id
+       * @param {string} attachmentId - the ID of the document you want to attach
```

Comment:
Tu peux préciser "the Drive ID", ça soulève tout les possibles doutes sur ce qui est attendu.